node/internal/util/inspect.ts

```diff
@@ -34,7 +34,7 @@ const ansi = new RegExp(ansiPattern, "g");
 /**
  * Returns the number of columns required to display the given string.
  */
-export function getStringWidth(str, removeControlChars = true) {
+export function getStringWidth(str: string, removeControlChars = true): number {
```

Comment:
I'm not in favor of changing this file to TypeScript - it was intentionally taken from the upstream as JS file so it's easier to maintain and there are no differences between Node's and our implementation. Why do you want to change it to TS?

Reply:
I mistakenly thought that type inconsistency would occur when defining `util.inspect` for `internal/util/inspect.js` 💦 .

>there are no differences between Node's and our implementation

If I define [`util.inspect`](https://github.com/nodejs/node/blob/86e976f1f4f9a506385dccec4d83409b5f55964b/lib/internal/util/inspect.js#L286) in `deno_std/node/internal/util/inspect.js`, differences arise Node's and our implementation because of [primordials](https://github.com/nodejs/node/blob/86e976f1f4f9a506385dccec4d83409b5f55964b/lib/internal/util/inspect.js#L80) and [internalBinding](https://github.com/nodejs/node/blob/86e976f1f4f9a506385dccec4d83409b5f55964b/lib/internal/util/inspect.js#L95). is it ok?

Reply:
@wafuwafu13 it is expected that primordials will be changed, but you can shim `internalBindings` by adding a module to `std/node/internal_binding/` directory. If possible let's keep it JS with only required changes for primordials.

Reply:
:+1:  I close this PR.